### DISCUSSION TITLE
Output Feed Icon not show

### DISCUSSION THREAD

**Discussion by melchionnaeu** [FreshRSS/FreshRSS#6524] [Q&A] [answered]
Why I don't have the icon for generate the output feed in my new installation?

![Schermata 2024-06-01 alle 18 35 03](https://github.com/FreshRSS/FreshRSS/assets/37714897/497a63fb-e5c3-45e6-99eb-2eae0b25838a)

**Comment by Alkarex** [comment, marked as answer]
Hello, 
From [FreshRSS 1.24.0 change log](https://github.com/FreshRSS/FreshRSS/blob/1.24.0/CHANGELOG.md):
* New [*shareable user query*](https://freshrss.github.io/FreshRSS/en/users/user_queries.html#share-your-user-queries) mechanism to share list of articles by HTML, RSS, OPML [#6052](https://github.com/FreshRSS/FreshRSS/pull/6052)
    * Deprecates RSS sharing with master token

P.S. Although the old icon is removed from the UI, the underlying link is still declared in the HTML headers and can be found via e.g. a Firefox extension such as [Want My RSS](https://addons.mozilla.org/firefox/addon/want-my-rss/).

**Reply by melchionnaeu** [reply]
Thank you!
